tests/Build/Changelog/ChangelogBuilderTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace Aws\Test\Build\Changelog;
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+class ChangelogBuilderTest extends \PHPUnit_Framework_TestCase
+{
+	/**
+	 * @covers ChangelogBuilder
+	 *
+	 */
+
+	private $RESOURCE_DIR= "tests/Build/Changelog/resources";
+
+	private function getChangelogBuilder()
+	{
+		return new ChangelogBuilder();
+	}
+
+	public function testreadChangelogNoDirectory()
+	{
+		$this->setExpectedException(\Exception::class);
+		$obj =$this->getChangelogBuilder();
+		$result = $obj->readChangelog("wrong-folder");
+	}
+
+	public function testreadChangelogNoReleaseNotes()
```

Comment:
This might be same, but could you make sure that those function naming also follows camelCases?